pypeman/tests/test_channel.py

```diff
@@ -16,6 +16,18 @@
 from pypeman.tests.common import TstNode
 
 
+def raise_dropped(msg):
```

Comment:
doc string:
show how and why this function would be used instead of directly raising the exception.